nautobot/core/tests/test_api.py

```diff
@@ -810,6 +817,9 @@ def test_descending_sort(self):
 class NewUIGetMenuAPIViewTestCase(testing.APITestCase):
     def test_get_menu(self):
         """Asset response from new ui nav menu api returns a well formatted registry["new_ui_nav_menu"] expected by nautobot-ui."""
+        self.user.is_superuser = True
+        self.user.save()
+
         url = reverse("ui-api:get-menu")
         response = self.client.get(url, **self.header)
```

Comment:
Shouldn't this test case require the user to be logged in? Do we have a bug here?

Reply:
User access token is provided in `self.header`